### PR TITLE
Update docs with new `rad init` behavior

### PR DESCRIPTION
**Pull request by AaronCrawfis**
Updates `rad init` commands based on new behavior introduced in https://github.com/project-radius/radius/pull/5665

### CODE REVIEW

File: docs/content/operations/environments/workspaces/index.md [hunk 1]
```diff
@@ -108,16 +108,16 @@ When you have multiple environments initialized for different purposes like stag
...
     ```sh 
-    rad init
+    rad init --full
     ```
     Name your environment 'staging' when prompted.
 1. Create another workspace named `production`:
 
     ```sh 
-    rad init kubernetes
+    rad init
```
Comment (AaronCrawfis) [inline docs/content/operations/environments/workspaces/index.md]: I'm realizing now this is incorrect. Will fix

Comment (willtsai) [reply docs/content/operations/environments/workspaces/index.md]: will take a look after you fix

Comment (AaronCrawfis) [reply docs/content/operations/environments/workspaces/index.md]: Updated, pls review

File: docs/content/operations/environments/workspaces/index.md [hunk 2]
```diff
@@ -97,47 +97,55 @@ Visit the [`config.yaml` reference docs]({{< ref config >}}) to learn about work
...
           kind: kubernetes
-        environment: /planes/radius/local/resourcegroups/staging
-        /providers/applications.core/environments/staging
-        scope: /planes/radius/local/resourceGroups/staging
+        environment: /planes/radius/local/resourcegroups/myworkspace
+        /providers/applications.core/environments/myworkspace
+        scope: /planes/radius/local/resourceGroups/myworkspace
     ```
-1. You can now deploy applications to both staging and prod using [`rad deploy`]({{< ref rad_deploy >}}), specifying the `-w` flag:
+1. You can now deploy applications to both myworkspace and prod using [`rad deploy`]({{< ref rad_deploy >}}), specifying the `-w` flag.
```
Comment (AaronCrawfis) [inline docs/content/operations/environments/workspaces/index.md]:
```suggestion
1. You can now deploy applications to both myworkspace and yourworkspace using [`rad deploy`]({{< ref rad_deploy >}}), specifying the `-w` flag.
```

**Review by willtsai — APPROVED** [review]

**Pull request merged**